src/pipecat/processors/frame_processor.py

```diff
@@ -620,6 +632,41 @@ async def push_frame(self, frame: Frame, direction: FrameDirection = FrameDirect
 
         await self.__internal_push_frame(frame, direction)
 
+        if isinstance(frame, InterruptionFrame):
+            self._wait_interruption_event.set()
+
+    async def push_interruption_task_frame_and_wait(self, timeout: Optional[float] = None):
```

Comment:
When would someone need to/want to set a timeout here? We're pretty much guaranteed to receive the `InterruptionFrame`, right, and in a fairly timely manner?

Wondering if we can start simpler and then only introduce the timeout if the need arises.